program/tests/context.rs

```diff
@@ -881,6 +884,89 @@ impl Context {
             .expect("Failed to call StakeDeposit on Solido instance.")
     }
 
+    /// Unstake from the validator at `self.validator`.
+    pub async fn try_unstake(&mut self, amount: Lamports) -> transport::Result<()> {
+        // Where the new stake will live.
+        let solido = self.get_solido().await;
+        // solido.validators.entries[0].entry.get
+        let validator = solido
+            .validators
+            .get(&self.validator.as_ref().unwrap().vote_account)
+            .unwrap();
+
+        let (source_stake_account, _) = Validator::find_stake_account_address(
+            &id(),
+            &self.solido.pubkey(),
+            &validator.pubkey,
+            validator.entry.stake_seeds.stake_accounts_seed_begin,
+        );
+        let (destination_stake_account, _) = Validator::find_unstake_account_address(
+            &id(),
+            &self.solido.pubkey(),
+            &validator.pubkey,
+            validator.entry.unstake_seeds.stake_accounts_seed_begin,
```

Comment:
Shouldn’t this be the end? Initially `begin = end = 0`, but after one unstake, this will matter.